table.go

```diff
@@ -0,0 +1,208 @@
+			row = append(row, ci.MAC)
+			row = append(row, metadata.ByteToString(archiveSizes.checkpointSize))
+			row = append(row, metadata.ByteToString(archiveSizes.rootFsDiffTarSize))
+		}
+
+		table.Append(row)
+	}
+
+	table.SetHeader(header)
+	table.SetAutoMergeCells(false)
+	table.SetRowLine(true)
+	table.Render()
+
+	// If there is only one checkpoint to show, check the mounts and stats flags
+	if len(tasks) == 1 {
+		if mounts {
+			renderMounts(specDump)
+		}
+
+		if stats {
+			// Get dump statistics with crit
+			dumpStats, err := crit.GetDumpStats(tasks[0].outputDir)
+			if err != nil {
+				return fmt.Errorf("failed to get dump statistics: %w", err)
+			}
+
+			renderDumpStats(dumpStats)
+		}
+
+		if psTree {
+			// The image files reside in a subdirectory called "checkpoint"
+			c := crit.New("", "", filepath.Join(tasks[0].outputDir, "checkpoint"), false, false)
+			// Get process tree with CRIT
+			psTree, err := c.ExplorePs()
+			if err != nil {
+				return fmt.Errorf("failed to get process tree: %w", err)
+			}
+
+			tree := renderPsTree(psTree, ci.Name)
+
+			fmt.Print("\nProcess tree\n\n")
+			fmt.Println(tree.String())
+		}
+	}
+
+	return nil
+}
+
+func renderMounts(specDump *spec.Spec) {
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetHeader([]string{
+		"Destination",
+		"Type",
+		"Source",
+	})
+	// Get overview of mounts from spec.dump
+	for _, data := range specDump.Mounts {
+		table.Append([]string{
+			data.Destination,
+			data.Type,
+			func() string {
+				if fullPaths {
+					return data.Source
+				}
+				return shortenPath(data.Source)
+			}(),
+		})
+	}
+	fmt.Println("\nOverview of Mounts")
+	table.Render()
+}
+
+func renderDumpStats(dumpStats *images.DumpStatsEntry) {
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetHeader([]string{
+		"Freezing Time",
+		"Frozen Time",
+		"Memdump Time",
+		"Memwrite Time",
+		"Pages Scanned",
+		"Pages Written",
+	})
+	table.Append([]string{
+		fmt.Sprintf("%d us", dumpStats.GetFreezingTime()),
+		fmt.Sprintf("%d us", dumpStats.GetFrozenTime()),
+		fmt.Sprintf("%d us", dumpStats.GetMemdumpTime()),
+		fmt.Sprintf("%d us", dumpStats.GetMemwriteTime()),
+		fmt.Sprintf("%d", dumpStats.GetPagesScanned()),
+		fmt.Sprintf("%d", dumpStats.GetPagesWritten()),
+	})
+	fmt.Println("\nCRIU dump statistics")
+	table.Render()
+}
+
+func renderPsTree(psTree *crit.PsTree, containerName string) treeprint.Tree {
+	var tree treeprint.Tree
+	if containerName == "" {
+		containerName = "Container"
+	}
+	tree = treeprint.NewWithRoot(containerName)
+	// processNodes is a recursive function to create
+	// a new branch for each process and add its child
+	// processes as child nodes of the branch.
+	var processNodes func(treeprint.Tree, *crit.PsTree)
+	processNodes = func(tree treeprint.Tree, root *crit.PsTree) {
+		node := tree.AddMetaBranch(root.PId, root.Comm)
+		for _, child := range root.Children {
+			processNodes(node, child)
+		}
+	}
+
+	processNodes(tree, psTree)
+
+	return tree
```

Comment:
I think it would make sense to move the printing logic here to keep it consistent with the other rendering functions